src/index.js

```diff
@@ -528,6 +566,25 @@ async function start() {
     await msg.unpin();
   });
 
+  // Gets debug event and some rate limit events in the shape of a 429 message (ie. set channel topic)
+  discordClient.on("debug", (...args) => {
+    if (VERBOSE) {
+      console.log("Debug: ", ...args);
+    }
+  });
+
+  // Gets rate limit events for some (not all) routes (ie. get channel pins)
+  discordClient.on("rateLimit", info => {
+    if (VERBOSE) {
```

Comment:
Should we log these on normal log levels instead? I don't think VERBOSE is enabled on production servers by default.

Reply:
Done.